src/aead/aes_gcm.rs

```diff
@@ -129,4 +127,56 @@ mod tests {
         test_aead(&aead::AES_256_GCM,
                   "crypto/cipher/test/aes_256_gcm_tests.txt");
     }
+
+    fn convert_block(data: &[u8]) -> [u8; AES_BLOCK_SIZE] {
+        let mut buf = [0u8; AES_BLOCK_SIZE];
+        buf.clone_from_slice(data);
+        buf
+    }
+
+    #[test]
+    pub fn test_aes() {
+        test::from_file("src/aead/aes_tests.txt", |section, test_case| {
+            assert_eq!(section, "");
+            let key = test_case.consume_bytes("Key");
+            let input = convert_block(&test_case.consume_bytes("Input"));
+            let output = convert_block(&test_case.consume_bytes("Output"));
```

Comment:
Instead of doing this, let's do something like this:

```
let input = test_case.consume_bytes("Input");
let input = slice_as_array_ref!(&input, AES_BLOCK_SIZE);
let output = test_case.consume_bytes("Output");
let output = slice_as_array_ref!(&output, AES_BLOCK_SIZE);
```